Source/Blazorise/Components/Table/Table.razor.cs

```diff
@@ -70,6 +71,7 @@ protected override async Task OnAfterRenderAsync( bool firstRender )
     protected override void BuildClasses( ClassBuilder builder )
     {
         builder.Append( "b-table" );
+        builder.Append( "b-table-mobile", ResponsiveMode == TableResponsiveMode.Mobile );
```

Comment:
Make "b-table-mobile" as part of `IClassProvider`. And maybe name it accordoing to each provider naming scheme.

Also make it receive the TableResponsiveMode enum in case we want to resolve different modes in the future. Default will return null.